oracle/provider/websocket_controller.go

```diff
@@ -193,9 +193,8 @@ func (conn *WebsocketConnection) SendJSON(msg interface{}) error {
 	defer conn.mtx.Unlock()
 
 	if conn.client == nil {
-		return fmt.Errorf("unable to send JSON on a closed connection")
+		return nil
```

Comment:
Should we at least log a debug warning here when the json client is closed?

Reply:
We already log an error when the socket is closed. This is always redundant. Either there already was a previous error or the websocket is being intentionally closed and we don't need to send any of the queued up messages.

Reply:
< 3